README.md

```diff
@@ -54,25 +54,43 @@ sfdx force:org:open
 
 ## Post scratch setup
 
-As some metadata have poor support for packaging and metadata deployment there are a few manual steps to perform to be able to test the chat solution:
-
-1. Create a LiveChatButton
-    - Go to setup > .. Chat Buttons & Invitations. Create a new chat button with omni-channel routing connected to the scratch chat queue
-2. Create a embedded service deployment and configuration
-    - Go to setup > .. Embedded Service Deployments and click new deployment, chose embedded chat and and deploy to default experience site
-    - Under chat settings click start and and then save with the prefilled config.
-3. Go to the experience site scratch_innboks and into builder. Find the embedded service chat component and update the chat deployment to the one newly created.
-4. In the builder, open settings > Security and privacy and Enable relaxed csp (if not already enabled). Then under the CSP Errors section allow the two sites that have been blocked from the live agent endpoints.
-5. Navigate to the workspace of scratch-innboks. The easiest way to get there is the hamburger in the top left of the builder. Go to Administration > Members and add customer profile Scratch Community Profile and save.
-6. Go to setup > Permission sets > Scratch Permission set and add access to the service presence statuses needed for chat.
-7. Run this command in the terminal
+As some metadata have poor support for packaging and metadata deployment there are a few manual steps to perform to be able to test the chat solution.
 
-```
-npm run scratchSetup
-```
+1.  -   Run this command in the terminal
+
+    ```
+    npm run scratchSetup
+    ```
+
+2.  Create a Messaging Channel
+
+    -   Go to Setup -> Messaging Settings -> Make sure Messaging is ON. Then press New Channel -> Messaging for In-App and Web. Under Omni-Channel Routing set Routing Type to Omni-Queue and assign Scratch Chat Queue. Make sure to activate the channel.
```

Comment:
"Make Sure to active the channel" e ganske utydelig, spesielt siden man må navigere seg tilbake og inn i channelen på nytt